tests/TcpConnectorTest.php

```diff
@@ -11,7 +11,7 @@
 
 class TcpConnectorTest extends TestCase
 {
-    const TIMEOUT = 0.1;
+    const TIMEOUT = 5.0;
```

Comment:
Purely for windows?

Reply:
This maximum timeout affects all platforms. It is very common to execute in a few milliseconds, the timeout is only used as an upper bound so the test will be cancelled in case something goes wrong. In particular, executing Windows in a VM means that timers are somewhat inaccurate and things sometimes take another second or two, but works just fine otherwise.

Reply:
👍